.github/workflows/automerge_plugin-only_prs.yml

```diff
@@ -0,0 +1,150 @@
+    runs-on: ubuntu-latest
+    needs: travis_success
+    if: |
+      ${{ needs.travissuccess.outputs.TRAVIS_OK == 'False' }} &&
+      contains( github.event.pull_request.labels.*.name, 'automerge-web')
+    steps:
+    - name: Parse user ID from PR title
+      run: |
+        echo "BS_UID="$(<<<${{ github.event.pull_request.title }} | sed -E 's/.*\(user:([^)]+)\).*/\1/'"" >> $GITHUB_ENV
+    - name: Build project
+      run: |
+        python -m pip install "."
+    - name: Get email from Brain-Score UID
+      run: |
+        echo "TO_EMAIL=$(python .github/workflows/call_endpoints.py get_email_from_uid '${{ env.BS_UID }}')" >> $GITHUB_ENV
+    - name: Send email to Brain-Score user
+      uses: dawidd6/action-send-mail@v3
+      with:
+        # Required mail server address:
+        server_address: smtp.gmail.com
+        # Server port, default 25:
+        server_port: 465
+        # Optional whether this connection use TLS (default is true if server_port is 465)
+        secure: true
+        # Optional (recommended) mail server username:
+        username: ${{secrets.MAIL_USERNAME}}
+        # Optional (recommended) mail server password:
+        password: ${{secrets.MAIL_PASSWORD}}
+        # Required mail subject:
+        subject: Brain-Score submission failed
+        # Required recipients' addresses:
+        to: ${{ env.BS_UID }}
+        # Required sender full name:
+        from: Brain-Score
+        # Optional plain body:
+        body: Your Brain-Score submission did not pass checks. Please review the test results and update the PR at ${{github.event.pull_request._links.self}}
+
+  checkchanges:
+    name: Check if PR only changes plugin files
+    runs-on: ubuntu-latest
+    needs: travis_success
+    if: ${{ needs.travissuccess.outputs.TRAVIS_OK == 'True' }}
+    outputs:
+      PLUGIN_INFO: ${{ steps.getpluginfo.outputs.PLUGIN_INFO }}
+      IS_PLUGIN_ONLY: ${{ steps.ispluginonly.outputs.IS_PLUGIN_ONLY }}
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Get changed files
+        uses: dorny/paths-filter@v2.11.1
+        id: filter
+        with:
+          list-files: shell
+          filters: |
+            changed:
+              - '**'
+      - name: Save changed files to env var
+        run: echo "CHANGED_FILES=${{ steps.filter.outputs.changed_files }}" >> $GITHUB_ENV
+      
+      - name: Parse changed files with python script
+        run: |
+          echo "PLUGIN_INFO=$(python .github/workflows/parse_changed_files.py '${{ env.CHANGED_FILES }}')" >> $GITHUB_ENV
+          
+      - name: Save plugin info to outputs
+        id: getpluginfo
+        run: |
+          echo "PLUGIN_INFO=$PLUGIN_INFO" >> $GITHUB_OUTPUT
+
+      - name: check if plugin-only
+        id: ispluginonly
+        run: |
+          echo "IS_PLUGIN_ONLY=$(jq -r '.is_plugin_only' <<< "$PLUGIN_INFO")" >> $GITHUB_OUTPUT
```

Comment:
The environment variable set by `parse_changed_files.py` is a dict that needs to be parsed into individual vars. This is legacy from before the automerge functionality was split out from the scoring functionality into two workflows. 

I'm guessing you'd prefer not spending the time to update it on this round. (I'm still leaning towards doing it anyways because that's nearly guaranteed to end up as permanently unaddressed tech debt otherwise 😁 )

Reply:
yes, I'm actually in favor of addressing this because it's going to continue throwing us off in the future otherwise.

Reply:
To be clear, it's still going to return a dict in the scoring workflow, so the `.` will remain there.